### PR TITLE
[ONNX]Update coverage info

### PR DESCRIPTION
**Pull request by zrphercule**
Right now we dont have coverage info of how many pytorch operators can be exported to onnx. This pr will add torch.nn operators to it, while later functional modules will be added as well.

### CODE REVIEW

File: test/onnx/export_onnx_tests_generator.py [hunk 1]
```diff
@@ -52,13 +52,44 @@ def gen_module(testcase):
...
+            unsupported.append(key)
+        elif (value == 3):
+            not_fully_supported.append(key)
+
+    def fun(info, l):
+        print(info)
+        for v in l:
+            print(v)
+
+    for info, l in [["{} Fully Supported Operators:".format(len(supported)), supported],
```
Comment (houseroad) [inline test/onnx/export_onnx_tests_generator.py]: add some comments to make it clear: what is supported/semi-supported/unsupported?

**Review by houseroad — COMMENTED** [review]
Overall looks good.

File: test/onnx/export_onnx_tests_generator.py (hunk 1 above)
Comment (zrphercule) [reply test/onnx/export_onnx_tests_generator.py]: Sure, btw I updated it in quip.

**Comment by zrphercule** [comment]
@houseroad a stamp plz? =w=

**Review by houseroad — APPROVED** [review]
Looks good, lint?

**Pull request closed without merging**